CALLS.md

```diff
@@ -0,0 +1,62 @@
+Calls
+=====
+
+The HTTP client’s job is to accept your request and produce its response. This is simple in theory but it gets tricky in practice.
+
+#### [Requests](http://square.github.io/okhttp/3.x/okhttp/okhttp3/Request.html)
```

Comment:
why 3.x?

Reply:
I’m gonna change ’em to 4.x after I merge this into our 3.x branch.